basics/installation/environments/docker.md

```diff
@@ -339,14 +339,18 @@ services:
       image: phpmyadmin/phpmyadmin
       container_name: phpmyadmin
       links:
-        - some-mysql
+        - mysql
       environment:
         PMA_HOST: some-mysql
         PMA_PORT: 3306
         PMA_ARBITRARY: 1
+        PMA_USER: root
+        PMA_PASSWORD: admin
```

Comment:
@SharakPL is this the intent of having the host name and port set here? see comment on the commit, at current it isnt used unless the the PMA_HOST and PMA_PASSWORD is set, I think this might have been the intent and if not could be a great add to the example to assist new users.